scripts/fig_2b-qc_plots.R

```diff
@@ -0,0 +1,278 @@
+# This script is used to generate simplified version of all the plots in the main QC report 
+
+# load project
+renv::load()
+
+library(SingleCellExperiment)
+library(dplyr)
+library(ggplot2)
+
+# Set default ggplot theme
+theme_set(
+  theme_classic() +
+    theme(
+      #plot.margin = margin(rep(20, 4)),
+      strip.background = element_rect(fill = "transparent"),
+      # no background box for legend
+      legend.background = element_blank(),
+      # no axis ticks or labels
+      axis.ticks = element_blank(),
+      axis.text = element_blank(),
```

Comment:
Minor thought because some doubled line drawing was bothering me a touch.
```suggestion
      axis.line = element_blank(),
      axis.ticks = element_blank(),
      axis.text = element_blank(),
```